spec/url_helper_spec.coffee

```diff
@@ -163,6 +163,20 @@ describe 'URLHelper', ->
 
       expect(@subject.getParamsFromUrl(url)).to.deep.equal(params)
 
+    context 'when contains a malformed param without a key', ->
```

Comment:
I d propose to add some more complex cases with multiple params with empty name

Reply:
agreed, b1d9ef1fc60f57ac1557d7c63949443c1dd4e559